app/retail/templates/press_page.html

```diff
@@ -0,0 +1,200 @@
+                    <div class="grid-item"><blockquote class="twitter-tweet" data-cards="hidden" data-lang="en"><p lang="en" dir="ltr">The day of the Ethereal Hackathon has arrived! Please check out the Hackathon Explorer to see 18 prizes (&amp; growing!) which you can build towards at the event, and join the conversation in Discord! 🎉<a href="https://t.co/macD1KCkz9">https://t.co/macD1KCkz9</a></p>&mdash; Gitcoin (@gitcoin) <a href="https://twitter.com/gitcoin/status/1117811846684250112?ref_src=twsrc%5Etfw">April 15, 2019</a></blockquote></div>
+                    <div class="grid-item"><blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">The day of the Ethereal Hackathon has arrived! Please check out the Hackathon Explorer to see 18 prizes (&amp; growing!) which you can build towards at the event, and join the conversation in Discord! 🎉<a href="https://t.co/macD1KCkz9">https://t.co/macD1KCkz9</a></p>&mdash; Gitcoin (@gitcoin) <a href="https://twitter.com/gitcoin/status/1117811846684250112?ref_src=twsrc%5Etfw">April 15, 2019</a></blockquote></div>
+                    <div class="grid-item"><blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">The day of the Ethereal Hackathon has arrived! Please check out the Hackathon Explorer to see 18 prizes (&amp; growing!) which you can build towards at the event, and join the conversation in Discord! 🎉<a href="https://t.co/macD1KCkz9">https://t.co/macD1KCkz9</a></p>&mdash; Gitcoin (@gitcoin) <a href="https://twitter.com/gitcoin/status/1117811846684250112?ref_src=twsrc%5Etfw">April 15, 2019</a></blockquote></div>
+                    <div class="grid-item"><blockquote class="twitter-tweet" data-cards="hidden" data-lang="en"><p lang="en" dir="ltr">The day of the Ethereal Hackathon has arrived! Please check out the Hackathon Explorer to see 18 prizes (&amp; growing!) which you can build towards at the event, and join the conversation in Discord! 🎉<a href="https://t.co/macD1KCkz9">https://t.co/macD1KCkz9</a></p>&mdash; Gitcoin (@gitcoin) <a href="https://twitter.com/gitcoin/status/1117811846684250112?ref_src=twsrc%5Etfw">April 15, 2019</a></blockquote></div>
+                    <div class="grid-item"><blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">The day of the Ethereal Hackathon has arrived! Please check out the Hackathon Explorer to see 18 prizes (&amp; growing!) which you can build towards at the event, and join the conversation in Discord! 🎉<a href="https://t.co/macD1KCkz9">https://t.co/macD1KCkz9</a></p>&mdash; Gitcoin (@gitcoin) <a href="https://twitter.com/gitcoin/status/1117811846684250112?ref_src=twsrc%5Etfw">April 15, 2019</a></blockquote></div>
```

Comment:
^ could we clean these up and get the from views.py as a list and simply loop through them ?

Reply:
The tweets for now are different but will soon be dynamic